### ISSUE TITLE
Clean up / refactor subsets

### ISSUE THREAD

**Issue by matuskalas** [edamontology/edamontology#418]
This becomes important now that the work on interdisciplinary EDAM / EDAM specialisations has started.

- [ ] Encoding of subsets need a review, ideally with dereferenceable URIs or even better cool URLs
- [ ] Sub-ontology subsets: every data_|format_|operation_|topic_ concept should have ***data|format|operation|topic*** subset
- [ ] Domain subsets: everything in the mainline EDAM needs subset ***bio***
- [ ] Generic, higher-level concepts need an additional subset ***generic***

Bonus: In the mid-term future, we might also need a subset ***interdisciplinary***, at least for topics and generic concepts. Needs some experiments first, so that one for later...

**Comment by joncison** [comment]
Nice!  Extra work for the validator, too.

**Comment by matuskalas** [comment]
Yes! Validator: https://github.com/edamontology/edamxpathvalidator/issues/14 Docs: https://github.com/edamontology/edamontologyDocs/issues/28

**Comment by joncison** [comment]
@matuskalas could you please just confirm you're still happy with the proposal here.  I'm adding this to the next milestone.

**Comment by joncison** [comment]
@matuskalas could you pls. clarify ... do you want _**bio**_ to replace current _**edam**_ subset, or (somehow) in addition to it?

if you reply quick the change can make it into EDAM 1.25

Thanks!

**Comment by joncison** [comment]
nudge-nudge @matuskalas  - please take a look at above thread, perhaps action can be taken in next realease?
